[quant][fix] Fix quantization for qat.ConvBnReLU1d

jerryzh168 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#48059 [quant][fix] Fix quantization for qat.ConvBnReLU1d**

Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D25006388](https://our.internmc.facebook.com/intern/diff/D25006388)

jerryzh168 committed:
[quant][fix] Fix quantization for qat.ConvBnReLU1d

Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][fix] Fix quantization for qat.ConvBnReLU1d"


Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D25006388](https://our.internmc.facebook.com/intern/diff/D25006388)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][fix] Fix quantization for qat.ConvBnReLU1d"


Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D25006388](https://our.internmc.facebook.com/intern/diff/D25006388)

[ghstack-poisoned]

vkuzo reviewed (approved):
lg, should we add a test to cover for the future?

jerryzh168 commented:
> lg, should we add a test to cover for the future?

This code is used by both graph and eager. Currently we have test to compare between fx graph mode and eager. but we actually didn't verify whether the result of eager is correct or not in general (maybe verified by training e2e model and seeing the result makes sense), maybe we can add a new set of tests for that in the future to test whether eager mode or graph mode quantizes each float op correctly if this is needed.

jerryzh168 committed:
Update on "[quant][fix] Fix quantization for qat.ConvBnReLU1d"


Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D25006388](https://our.internmc.facebook.com/intern/diff/D25006388)

[ghstack-poisoned]
